### DISCUSSION TITLE
为什么要移除buildLevel的warning选项？

### DISCUSSION THREAD

**Discussion by OEOTYAN** [xmake-io/xmake-vscode#251] [Q&A] [answered]
2.2.9 移除了Build Output Level的warning，这是为什么呢？

**Comment by waruqi** [comment, marked as answer]
下个版本 xmake 会默认开启 warning ，没必要再加这个了。

https://github.com/xmake-io/xmake/discussions/4642
https://github.com/xmake-io/xmake/issues/4473
